### ISSUE TITLE
Format Numbers in Output

### ISSUE THREAD

**Issue by djk447** [jakob/Postico#261]
I'd really love a setting I can toggle, sort of like \pset numericlocale in psql that would allow me to format numbers with commas so I can read large ones. Thanks!

**Comment by jakob** [comment]
I've considered displaying numbers in the system locale. The problem with that is that numbers would be displayed differently than in the SQL Query Editor. (This is especially problematic with `.` vs `,` as decimal separator)

Also, the number format should probably be configurable by column, right? When using the system locale, I saw an issue where eg. a year would be displayed as `2 016`, which looks really wrong.

What kind of format would you like to see?

**Comment by djk447** [comment]
Honestly, I'd follow what psql does with the numericlocale setting. Aside from that, isn't the type of the column available somewhere? Wouldn't only applying it to numeric, int, float etc types work? 
This is the psql output:  2016-07-10 |             | f                   |   125,284 |   226,887,500 for instance.

**Comment by djk447** [comment]
(Or-how hard would it be to enable psql commands generally? That would be incredibly helpful...)

**Comment by jakob** [comment]
Adding support for psql commands would have to be done one at a time -- I'd have to reimplement everything.

Also, there's a fundamental difference; psql has line based input, whereas Postico allows editing a big buffer of text at once. So even if I add support for psql commands, there's bound to be confusion, and people will start executing psql scripts, which might not work the same in Postico....

**Comment by djk447** [comment]
Yeah, I guess I was thinking that the easiest thing there would just be to treat the big buffer of text as a sql file, and then when you hit execute, it just runs it as if you were running a sql file through psql. I guess i mean that the single line buffer part of it doesn't seem to be a huge issue here, but I totally get it if it's problematic. I get the sense you're connecting directly through libpq or something, and not using a command line client, which would def mean some sort of re-implementation. But you might look at how psql deals with the numeric locale stuff for hints on how it handles the problems you mention. It seems pretty good at it. 

Having some of the other psql features like tab complete, \set & \gset  variables and various other settings and the like would really be amazing. I already like Postico a lot, but that would make it really killer so if it could run basically on top of psql....that'd be interesting at least. Or perhaps the easier implementation of this would be to have a small psql shell you could open with the same connection information etc. Anyway...possibly pie in the sky, but thought I'd ask!
